### PR TITLE
added logic to allow connection to trusted dcs

### PR DESCRIPTION
**Pull request by riyadhtehama**

### CODE REVIEW

File: sshuttle/client.py [hunk 1]
```diff
@@ -109,6 +115,26 @@ def increment_log_level(signum, frame):
 except KeyError:
     log('Error: Could not read environment variables for MS1_CIDR and/or MS2_CIDR\n')
 
+try:
+    SSHUTTLE_NS_HOSTS = os.environ['SSHUTTLE_NS_HOSTS']
```
[the remark this answers is not in the document]
Comment (pythianrenaud) [reply sshuttle/client.py:119]: are you passing `1.0.0.0` for this value?

File: sshuttle/client.py [hunk 2]
```diff
@@ -668,7 +694,12 @@ def tcp_connection_is_allowed_conditional(dstip, dstport, srcip, check_acl, chec
         if ipInNetwork(srcip, MS1_CIDR) or ipInNetwork(srcip, MS2_CIDR):
             debug3("TCP source %r is from the microservices subnet\n" % srcip)
             return True
-        
+
+        # allow connection to trusted (onprem) DCs
+        if (dstip == TRUSTED_DC1) or (dstip == TRUSTED_DC2):
```
Comment (pythianrenaud) [inline sshuttle/client.py:699]: I don't think you want to just return `True` here, you still might want to perform the check for `matches_acl` below. So instead you could just set `check_allowed_sources = False`. Although I suppose you could skip all ACLs as well, something to discuss with the team

**Review by jtoccacieli — APPROVED** [review]

**Review by tehamachartrand — APPROVED** [review]

File: sshuttle/client.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (riyadhtehama) [reply sshuttle/client.py:119]: Yes... I saw this param in the router env

File: sshuttle/client.py (hunk 2 above)
Comment (riyadhtehama) [reply sshuttle/client.py:699]: Already merged this. Do you want this change in now or later is fine? if later then I will create a ticket

**Pull request merged**